napari/layers/_tests/test_serialize.py

```diff
@@ -8,20 +8,32 @@
     count_warning_events,
     layer_test_data,
 )
+from napari.layers import Layer
 
 
-@pytest.mark.parametrize(('Layer', 'data', 'ndim'), layer_test_data)
-def test_attrs_arrays(Layer, data, ndim):
+@pytest.mark.parametrize(('LayerType', 'data', 'ndim'), layer_test_data)
```

Comment:
The name change here helps distinguish with the `Layer` type, which I introduced here to make the test a little more clear. This change does not need to occur in this PR.